druid-lookups/src/main/java/com/yahoo/maha/maha_druid_lookups/query/lookup/namespace/RocksDBExtractionNamespace.java

```diff
@@ -62,6 +68,16 @@ public RocksDBExtractionNamespace(@NotNull @JsonProperty(value = "namespace", re
         this.lookupAuditingEnabled = lookupAuditingEnabled;
         this.lookupName = lookupName;
         this.tsColumn = tsColumn;
+
+        try {
+            if(StringUtils.isNotBlank(cacheActionRunner) && Objects.nonNull(Class.forName(cacheActionRunner)))
+                this.cacheActionRunner = cacheActionRunner;
+            else {
+                this.cacheActionRunner = "";
+            }
+        } catch (Exception e) {
+            this.cacheActionRunner = "";
```

Comment:
Let's log this exception. An exception here would occur only if the given class is invalid or can't be found, which should not be ignored. In the current flow, it will throw a NullPointerException since tryResetRunnerOrLog method doesn't handle the case of empty string.

Reply:
Actually let's just surface this exception, the application shouldn't start if the desired class isn't present.